src/lib/qdl.js

```diff
@@ -24,31 +24,93 @@ async function getExecutable() {
 	return path.join(tmpDir, 'qdl' + (archName === 'win32' ? '.exe' : ''));
 }
 
-/**
- */
-async function run({ files, includeDir, updateFolder, zip, verbose, ui }) {
-	const qdl = await getExecutable();
-
-	const qdlArgs = [
-		'--storage',
-		TACHYON_STORAGE_TYPE,
+async function run({ files, includeDir, updateFolder, zip, ui, outputLogFile }) {
```

Comment:
My main feedback is to break this function into smaller functions to make sense of it.